File: 1441-build-an-array-with-stack-operations/1441-build-an-array-with-stack-operations.py
class Solution:
    def buildArray(self, target: List[int], n: int) -> List[str]:
        #List to save operations
        Operations = []
        current = 1
        #Iterate through array
        for num in target:
            #If number is on target Push
            #if number not on target Push Pop
            while current < num:
                Operations.append("Push")
                Operations.append("Pop")
                current += 1

            Operations.append("Push")
            current += 1
        return Operations

File: 1441-build-an-array-with-stack-operations/1441-build-an-array-with-stack-operations.java
class Solution {
    public List<String> buildArray(int[] target, int n) {
        List<String> Operations = new ArrayList<>();
        int current = 1;

        for (int num: target){
            while (current < num){
                Operations.add("Push");
                Operations.add("Pop");
                current++;
            }
            Operations.add("Push");
            current++;
        }

        return Operations;
    }
}